presto-native-execution/presto_cpp/main/types/PrestoToVeloxConnector.cpp

```diff
@@ -77,6 +77,16 @@ dwio::common::FileFormat toVeloxFileFormat(
     } else if (format.serDe == "org.apache.hive.hcatalog.data.JsonSerDe") {
       return dwio::common::FileFormat::JSON;
     }
+  } else if (
+      format.inputFormat ==
+      "org.apache.hadoop.hive.ql.io.SymlinkTextInputFormat") {
+    if (format.serDe ==
+        "org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe") {
+      return dwio::common::FileFormat::PARQUET;
+    } else {
+      VELOX_UNSUPPORTED(
```

Comment:
Do you really need this else ? You would hit the VELOX_UNSUPPORTED message on line 90 since there is no return path before.

Reply:
I too think, we don't really need have to add Else in here since any unsupported case would be covered at last.

This was added as part of review [comment](https://github.com/prestodb/presto/pull/23536#discussion_r1757061868) 
@majetideepak Should we take out this, WDYT?

Reply:
I did not see the default. We can remove it from here.